src/components/CheckStatus.js

```diff
@@ -1,7 +1,19 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
-const CheckStatus = () => (
-  <button type="button">CheckStatus</button>
-);
+const CheckStatus = () => {
```

Comment:
- [ ] I kindly suggest that you make sure your categories component displays a button that returns "Under Construction" when it is pressed. See video below for more details:

https://www.loom.com/share/ede984992eb546baa9ff672d674a19b2